parlai/tasks/empathetic_dialogues/agents.py

```diff
@@ -4,57 +4,28 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Optional
-from parlai.core.params import ParlaiParser
-from parlai.core.opt import Opt
 import os
-from typing import Any, List
-
-import numpy as np
+from typing import List, Optional
 
+import parlai.utils.logging as logging
 from parlai.utils.io import PathManager
 from parlai.core.message import Message
-from parlai.core.teachers import FixedDialogTeacher
+from parlai.core.params import ParlaiParser
+from parlai.core.opt import Opt
+from parlai.core.teachers import DialogTeacher
 from .build import build
 
 
 DEFAULT_TRAIN_EXPERIENCER_ONLY = False
-DEFAULT_REMOVE_POLITICAL_CONVOS = False
 
 
-class EmpatheticDialoguesTeacher(FixedDialogTeacher):
```

Comment:
ImportError: cannot import name 'EmpatheticDialoguesTeacher' from 'parlai.core.teachers'

Reply:
Can you paste the command that gives that error?